pkgs/by-name/sd/sdl_gamecontrollerdb/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "sdl_gamecontrollerdb";
+  version = "0-unstable-2024-11-05";
+
+  src = fetchFromGitHub {
+    owner = "mdqinc";
+    repo = "SDL_GameControllerDB";
+    rev = "cfc2bffe0ad29fea2bec7a0f4cb19dead5703ea8";
+    hash = "sha256-BEN0/q+7iLVmPpOO3i80nuLYpr7KiHd2ONxKmDIJuWU=";
+  };
+
+  dontBuild = true;
+  dontConfigure = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm644 gamecontrollerdb.txt -t $out/share
+    install -Dm644 LICENSE -t $out/share/licenses/${finalAttrs.pname}
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  meta = {
+    description = "A community sourced database of game controller mappings to be used with SDL2 and SDL3 Game Controller functionality";
```

```suggestion
    description = "Community sourced database of game controller mappings to be used with SDL2 and SDL3 Game Controller functionality";
```

Remove indefinite article from description.